sabnzbd/downloader.py

```diff
@@ -659,7 +659,7 @@ def run(self):
                         if bytes_received + sabnzbd.BPSMeter.bps > limit:
                             while sabnzbd.BPSMeter.bps > limit:
                                 time.sleep(0.01)
-                                sabnzbd.BPSMeter.update()
+                                sabnzbd.BPSMeter.update(force=True)
```

Comment:
Probably somewhere in the shutdown routine we should also call this with force=True, so all bytes are counted.

Reply:
Seems like we are forcing it everywhere now? Maybe we should have just done force is default true unless explicitly passed not to

Reply:
There's only 1 call to `update` that includes `server.id, bytes_received`, so there's no need for `force`. 
We can keep the old behavior and only skip in case there's a `server` supplied.
What do you think @puzzledsab ?

Reply:
@Safihre : Sorry, I don't understand what you are suggesting?

@thezoggy : As far as I can tell there are currently 2 calls with force and 4 without in downloader, and a few without force in bpsmeter.